src/base/kaldi-types.h

```diff
@@ -38,14 +38,26 @@ typedef float   BaseFloat;
 // we can do this a different way if some platform
 // we find in the future lacks stdint.h
 #include <stdint.h>
+#if _MSC_VER >= 1900
+#include <fst/types.h>
+#endif  // _MSCVER >= 1900
 
 namespace kaldi {
-typedef uint16_t        uint16;
-typedef uint32_t        uint32;
-typedef uint64_t        uint64;
-typedef int16_t         int16;
-typedef int32_t         int32;
-typedef int64_t         int64;
+#if _MSC_VER >= 1900
```

Comment:
Yes, we discussed making it like this, Yenda will change it.


On Sat, Mar 19, 2016 at 10:45 PM, Kirill Katsnelson <
notifications@github.com> wrote:

> In src/base/kaldi-types.h
> <https://github.com/kaldi-asr/kaldi/pull/600#discussion_r56758524>:
>
> >
> >  namespace kaldi {
> > -typedef uint16_t        uint16;
> > -typedef uint32_t        uint32;
> > -typedef uint64_t        uint64;
> > -typedef int16_t         int16;
> > -typedef int32_t         int32;
> > -typedef int64_t         int64;
> > +#if _MSC_VER >= 1900
>
> Maybe this should not even be conditional on MSC? The types would be
> correct either way, if my understanding is correct.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/600/files/c2e0c05d693409ab30dfad03c56003fdba424e47#r56758524>
>